ivy_tests/test_ivy/helpers.py

```diff
@@ -1845,11 +1858,13 @@ def array_values(
                     width=16,
                     exclude_min=exclude_min,
                     exclude_max=exclude_max,
-                ),
+                ).filter(lambda x: x <= -1 * limit or x >= limit),
```

Comment:
we should use the `min_val` and `max_val` [arguments](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.floats).

Reply:
@djl11 
I think if `min_val` and `max_val` are used, there could still be very small values generated because it generates from [`min_val` , `max_val`]. The way I understood is that we need values generated from [`min_val`, -limit] and [limit, `max_val`] where `limit` is calculated based on `small_value_safety_factor`.

Should `limit` be defined by `log(safety_factor)`?